services/github_service.py

```diff
@@ -1205,3 +1207,48 @@ def check_for_audit_log_new_members(self, since_date: str) -> list:
                 break
 
         return new_members
+    
+    @retries_github_rate_limit_exception_at_next_reset_once
+    def get_gha_minutes_used_for_organisation(self, organization) -> int:
+        logging.info(f"Getting all github actions minutes used for organization {organization}")
+        command = [
+            "gh", 
+            "api",
+            "-H", "Accept: application/vnd.github+json",
+            "-H", "X-GitHub-Api-Version: 2022-11-28",
+            f"/orgs/{organization}/settings/billing/actions"
```

Comment:
Ooof. I'm not sure I like the dependency of having ghcli.

Reply:
This would be better suited.
```
    url = f"https://api.github.com/orgs/{org_name}/settings/billing/actions"
    headers = {
        "Accept": "application/vnd.github+json",
        "Authorization": f"Bearer {your_token}",
        "X-GitHub-Api-Version": "2022-11-28"
    }
    
    response = requests.get(url, headers=headers)
    if response.ok:
        return response.json()
    else:
        return response.status_code, response.text
```

Reply:
In fact we do something similar here:
https://github.com/ministryofjustice/operations-engineering/blob/main/services/github_service.py#L849-L857